packages/controller-utils/src/constants.ts

```diff
@@ -126,12 +130,12 @@ export enum ApprovalType {
 }
 
 export const NETWORK_ID_TO_ETHERS_NETWORK_NAME_MAP: Record<
-  NetworkId,
+  InfuraNetworkId,
   NetworkType
 > = {
-  [NetworkId.goerli]: NetworkType.goerli,
-  [NetworkId.sepolia]: NetworkType.sepolia,
-  [NetworkId.mainnet]: NetworkType.mainnet,
-  [NetworkId['linea-goerli']]: NetworkType['linea-goerli'],
-  [NetworkId['linea-mainnet']]: NetworkType['linea-mainnet'],
+  [InfuraNetworkId.goerli]: NetworkType.goerli,
+  [InfuraNetworkId.sepolia]: NetworkType.sepolia,
+  [InfuraNetworkId.mainnet]: NetworkType.mainnet,
+  [InfuraNetworkId['linea-goerli']]: NetworkType['linea-goerli'],
+  [InfuraNetworkId['linea-mainnet']]: NetworkType['linea-mainnet'],
```

Comment:
What was the reason for this change? How does it relate to the CAIP-2 migration?

Reply:
Oh I see the name just changed for the type

Reply:
correct. unrelated to CAIP, but wanted to help disambiguate NetworkId and ChainId more by this naming. Happy to revert though